rcst/calc.py

```diff
@@ -32,3 +32,29 @@ def velocity_norm(present: Robot, previous: Robot, dt: float) -> float:
     vx = (present.x - previous.x) / dt
     vy = (present.y - previous.y) / dt
     return math.sqrt(vx**2 + vy**2)
+
+
+def calc_vector(x0, y0, x1, y1):
```

Comment:
型ヒントをつけたいです。（多分こうかな？）
```suggestion
def calc_vector(x0: float, y0: float, x1: float, y1: float) -> tuple[float, float]:
```